src/webgpu/api/operation/adapter/requestDevice.spec.ts

```diff
@@ -1,16 +1,106 @@
 export const description = `
 Tests for GPUAdapter.requestDevice.
 
-TODO:
-- descriptor is {null, undefined}
-- descriptor with the combinations of features
+Test all possible options to requestAdapter.
```

Comment:
I think this test is intended to be able `requestDevice`, but you've written a test for `requestAdapter`.

Your test should probably go in a new file `requestAdapter.spec.ts`.
Also, this description could probably be extended to say it runs a simple compute shader to test that the adapter works for something basic.